CONTRIBUTING.md

```diff
@@ -0,0 +1,50 @@
+# Contributing
+
+This repository includes a [GitHub Action][3] that builds the
+colorschemes once a week. This keeps the colorschemes up-to-date
+automatically.
+
+## Building
+
+### Dependencies
+
+- `>=0.2.0` [base16-builder-go][1]
+- golang `>=1.16` to build base16-builder-go
+
+### Usage for template editing
+
+1. Clone [base16-builder-go][1] somewhere on your system.
+1. Run `cd /path/to/base-builder-go && go build` to generate a binary:
+`/path/to/base-builder-go/base16-builder-go`
+1. Now execute the binary you generated while giving the `-template-dir`
+arg the path to `base16-xfce4-terminal` repository: `./base16-builder-go
+-template-dir ../base16-xfce4-terminal`
+
+Or the above steps represented in shell commands:
+
+```shell 
+cd /path/to/base16-xfce4-terminal/../ # This repos parent dir 
+git clone git@github.com:base16-project/base16-builder-go.git
+cd base16-builder-go
+go build ./base16-builder-go/base16-builder-go \
+  -template-dir ../base16-xfce4-terminal
+```
+
+### Usage for adding or editing a colorscheme
+
+If you want to add or edit a colorscheme but want to test it out, you
+simply need to pass in your local [base16-schemes][2] directory when
```

Comment:
```suggestion
need to pass in your local [base16-schemes] directory when
```

- nit on link style.
- `simply` is not recommended in various style guides [google](https://developers.google.com/style/word-list#simple), [gitlab](https://docs.gitlab.com/ee/development/documentation/styleguide/word_list.html#simply-simple)